packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -1082,6 +1088,32 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
     _syncAll(widget.gestures);
   }
 
+  @override
+  void didChangeAccessibilityFeatures() {
+    if (!widget.excludeFromSemantics)
+      _maybeUpdateSemanticsActionOverrides();
+  }
+
+  void _maybeUpdateSemanticsActionOverrides() {
+    final RenderSemanticsGestureHandler semanticsGestureHandler = _getSemanticsGestureHandler();
+    final Set<CustomSemanticsAction> overrides = <CustomSemanticsAction>{};
+    if (WidgetsBinding.instance.window.accessibilityFeatures.switchControl && semanticsGestureHandler.validActions != null) {
+      if (semanticsGestureHandler.validActions.contains(SemanticsAction.scrollUp)) {
+        overrides.add(const CustomSemanticsAction.overridingAction(label: 'Scroll Down', action: SemanticsAction.scrollUp));
```

Comment:
Hi @HansMuller or maybe @shihaohong , Is there a documentation i can follow if we want to localize these strings?